components/browser/icons/src/main/java/mozilla/components/browser/icons/processor/ResizingProcessor.kt

```diff
@@ -50,7 +50,6 @@ class ResizingProcessor(
             }
         }
 
-        originalBitmap.recycle()
```

Comment:
🤔  This seems important, I wonder why are we removing this line?

Reply:
Recycling the bitmap breaks the cache ☹️

Reply:
👍